src/platform/cannonlake/include/platform/lib/memory.h

```diff
@@ -421,6 +421,9 @@
 					SOF_STACK_TOTAL_SIZE)
 #define BOOT_LDR_STACK_SIZE		SOF_STACK_TOTAL_SIZE
 
+/* L1 init */
+#define L1_CACHE_PREFCTL_VALUE 0x0000
```

Comment:
Why was it added to cannonlake?

Reply:
Because it was hard coded in LPS files (with value for CNL)
TGL already have this in platform files